handlers/info.go

```diff
@@ -0,0 +1,43 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/openfaas/faas-provider/types"
+)
+
+const (
+	//NetesIdentifier identifier string for provider orchestration
+	NetesIdentifier = "kubernetes"
+	//NetesProvider name of the provider
+	NetesProvider = "faas-netes"
+)
+
+//MakeInfoHandler creates handler for /system/info endpoint
+func MakeInfoHandler(version, sha string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Body != nil {
+			defer r.Body.Close()
+		}
+
+		infoRequest := types.InfoRequest{
+			Orchestration: NetesIdentifier,
```

Comment:
Please can we call this VersionIdentifier / VersionProvider rather than "netes" as that isn't a word on its own? What do you think?

Reply:
I think we should change it. I will update that.